<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Danh sách sinh viên có mặt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>410</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Lưu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAttendance_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Điểm danh</string>
   </property>
  </widget>
  <widget class="QWidget" name="absent_2" native="true">
   <property name="geometry">
    <rect>
     <x>399</x>
     <y>379</y>
     <width>541</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStop_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Hoàn tất</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>360</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Danh sách sinh viên vắng mặt</string>
   </property>
  </widget>
  <widget class="QWidget" name="webcamWidget_2" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="present_2" native="true">
   <property name="geometry">
    <rect>
     <x>399</x>
     <y>29</y>
     <width>541</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblName_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
